internal/serverless/project.go

```diff
@@ -131,6 +138,54 @@ func (p *Project) DefaultFleetServerURL(kibanaClient *kibana.Client) (string, er
 	return fleetURL, nil
 }
 
+func (p *Project) AddLogstashFleetOutput(profile *profile.Profile, kibanaClient *kibana.Client) error {
+	logstashFleetOutput := kibana.FleetOutput{
+		Name:  "logstash-output",
+		ID:    FleetLogstashOutput,
+		Type:  "logstash",
+		Hosts: []string{"logstash:5044"},
```

Comment:
I think we could set here the SSL configuration directly, and then we would not need to update the output later.
At this point we already know the certificates and after the last changes they should not change.

Reply:
At this point the certificates are not created. They are created after `createProject` in `startLocalServices` step.